action.yml

```
@@ -30,5 +33,22 @@ runs:
       run: |
         VELO_PROJECT=${{ inputs.velo_project }}
         VELO_PROJECT="${VELO_PROJECT:-"${{ github.event.repository.name }}"}"
-        echo "copying to velo_project=${VELO_PROJECT}"
-        gsutil -m cp -r ${{ inputs.deploy_folder }}/* "gs://${{ inputs.google_cloud_storage_bucket_name }}/${VELO_PROJECT}/${{ github.sha }}/"
+        echo "::group::Figuring out artifact version"
+        VERSION="${{ inputs.version }}"
+
+        [ ! -z "$VERSION" ] && echo "::notice::Artifact version specified by user"
+
+        if [ -v "$VERSION" ] && ! command -v git; then
+          echo "::warning::Artifact version determined from github.sha due to missing 'git' binary"
+          export VERSION="${{ github.sha }}"
+        fi
```

```suggestion
        if [ -v "$VERSION" ] && ! command git 2>&1 >/dev/null; then
          echo "::error::Artifact version cannot be easily and reliably determined due to missing Git binary. Provide a 'version' or make sure 'git' is available."
          exit 1
        fi
```
@EeroPaukkonen 👍 / 👎?

---

Hehe, you're right - don't know how I ended up with -v as the test. I think that should be the culprit.